### PR TITLE
Cleanup and make consistent markdown files

### PR DESCRIPTION
**Pull request by philoserf**
- files had a different line length
- shell commands had leading `$` prompt
- add fenced code block language markers
- use native markup where some HTML was embedded
- fixup whitespace inline with community practice

---
This change is [Reviewable](https://reviewable.io/reviews/netzkern/butler/4)

### CODE REVIEW

**Comment by StarpTech** [comment]
Hi @philoserf thanks for your contribution.

File: README.md [hunk 1]
```diff
@@ -1,68 +1,79 @@
-<p align="center">
-<img src="/logo.png" alt="butler" style="max-width:100%;">
-</p>
+![butler](/logo.png)
```
Comment (StarpTech) [inline README.md]: The image should be centered.

File: README.md [hunk 2]
```diff
@@ -1,68 +1,79 @@
...
-## Documentation
+# Documentation
 
-* <a href="/docs/config.md"><code><b>Config</b></code></a>
-* <a href="/docs/templateSurveys.md"><code><b>Template Surveys</b></code></a>
-* <a href="/docs/templateSyntax.md"><code><b>Template Syntax</b></code></a>
-* <a href="/docs/gitHooks.md"><code><b>Git Hooks</b></code></a>
-* <a href="/docs/debugging.md"><code><b>Debugging</b></code></a>
-* <a href="#commands"><code><b>Commands</b></code></a>
+- [`**Config**`](/docs/config.md)
```
Comment (StarpTech) [inline README.md:37]: This looks wrong.

File: README.md [hunk 3]
```diff
@@ -1,68 +1,79 @@
...
-## License
+# License
 
 Licensed under [MIT](./LICENSE).
 
-### Credits
+## Credits
 
-<div>Icons made by <a href="http://www.freepik.com" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/" title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div>
+Icons made by [Freepikcom](http://www.freepik.com Freepik)
```
Comment (StarpTech) [inline README.md]: Some links are broken

Comment (philoserf) [reply README.md]: I will check my syntax.

File: README.md (hunk 1 above)
Comment (philoserf) [reply README.md]: If it must be then we need to keep the HTML. I will update.

Comment (StarpTech) [reply README.md]: Yep, it looks nicer 😄

File: README.md (hunk 2 above)
Comment (philoserf) [reply README.md:37]: the syntax is correct here

File: README.md [hunk 4]
```diff
@@ -2,67 +2,80 @@
...
-* <a href="#commands"><code><b>Commands</b></code></a>
+- [`**Config**`](/docs/config.md)
+- [`**Template Surveys**`](/docs/templateSurveys.md)
+- [`**Template Syntax**`](/docs/templateSyntax.md)
+- [`**Git Hooks**`](/docs/gitHooks.md)
+- [`**Debugging**`](/docs/debugging.md)
+- [`**Commands**`](#commands)
 
-## Commands
+# Commands
```
Comment (philoserf) [inline README.md]: wait. this was unintentional
fixing

**Comment by philoserf** [comment]
that should do it

File: README.md (hunk 2 above)
Comment (StarpTech) [reply README.md:37]: No, look at the preview the text should be bold.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -2,54 +2,61 @@
...
 
-This is an experiment and feedback is welcome! This document may also be subject to pull-requests or changes by contributors where you believe you have something valuable to add or change.
+This is an experiment and feedback is welcome! This document may also be
+subject to pull-requests or changes by contributors where you believe you have
+something valuable to add or change.
 
------------------------------------------
+--------------------------------------------------------------------------------
+
+[]()
```
Comment (StarpTech) [inline CONTRIBUTING.md:42]: What is this?

**Review by StarpTech — CHANGES_REQUESTED** [review]

**Comment by StarpTech** [comment]
Hi @philoserf due to many changes, I think this PR will be very hard to merge. Is it ok when I pick up the most important changes to master? After this, you can create a new one or leave it as it is. Thanks for your contribution.

**Comment by philoserf** [comment]
Will do. I will create a few smaller ones.

**Pull request closed without merging**